Makefile

```diff
@@ -41,7 +41,7 @@ clippy: pre-clippy
 	cargo clippy -- -A module_inception -A needless_pass_by_value -A cyclomatic_complexity \
 	    -A unreadable_literal -A should_implement_trait -A verbose_bit_mask \
 	    -A implicit_hasher -A large_enum_variant -A new_without_default_derive \
-	    -A neg_cmp_op_on_partial_ord
+	    -A neg_cmp_op_on_partial_ord -A cast_ptr_alignment
```

Comment:
It may be dangerous to allow this lint globally. Please see the [lint description](https://rust-lang-nursery.github.io/rust-clippy/master/index.html#cast_ptr_alignment).

You can do a `#[cfg_attr(feature = "cargo-clippy", allow(cast_ptr_alignment))]` to allow it only in the given case where you have determined it is safe.